src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/QuestionnaireResponseProcessing.tsx

```diff
@@ -87,14 +87,14 @@ const QuestionnaireResponseProcessing = (props: {
 
     return (
         <>
-            {questionnaire && questionnaire.name && (
+            {questionnaire && (
                 <div>
                     <SchemaView
                         questions={questions}
                         answers={answers}
                         questionnaireResource={questionnaireResource}
                         date={dayjs(props.schemaResponse.authored)}
-                        title={questionnaire.name}
+                        title={questionnaire.title ? questionnaire.title : 'Tittel mangler'}
```

Comment:
Perhaps but some form of styling on this, to accentuate the fact that the name is missing? For example italic, thoughts?

Reply:
This is only data sent to another component, but could add a check in schemaview. Personally I don't think it's needed because it's already informed that the scheme don't have a title in patientview (Rebekka's PR)